.circleci/config.yml

```diff
@@ -153,7 +148,7 @@ workflows:
             - browserstack
           filters:
             branches:
-              only: *releasable_branches
+              only: master
```

Comment:
FYI: This will be overwritten by https://github.com/LiveIntent/live-connect/pull/115. IMHO, it's better to leave alias, since we already have it.

Reply:
I think without the `releasable_branches` constant it might actually be cleaner. We can not use the constant for `release_major?` since it is only reserved for `master`. Thus in `release_major?` we would still use `master` and in all other steps we would use `releasable_branches` which for now only consist of `master`. That is confusing. I think we should re-introduce `releasable_branches` when really required. 

> FYI: This will be overwritten by https://github.com/LiveIntent/live-connect/pull/115.
Please do no overwrite everything. It needs to be merged properly.

Reply:
I will merge it manually and submit to review, yours truly CO